### ISSUE TITLE
https://mon-cadastre.fr/

### ISSUE THREAD

**Issue by peclet1248** [openstreetmap/tile-attribution#36]
### Is it maybe reported already?

- [X] I have searched the existing issues, it is not reported already

### Before you submit this form, please ensure the following criteria are met:

- [X] The website/app in question uses map tiles hosted by the OpenStreetMap foundation (https://tile.openstreetmap.org).
- [X] The website/app does not have proper attribution. (https://osmfoundation.org/wiki/Licence/Attribution_Guidelines)
- [X] The usage of the tiles is non-trivial. (Not a development site, not test code, not a personal project with negligible traffic, etc.)
- [ ] You have contacted the website/app owner or representative using the Love Letter model/template (https://osmfoundation.org/wiki/Licence/Attribution_Reminder_Templates) more than a week ago and no proper attribution has been added.
- [ ] Or you have made a good faith effort to find working contact information for the website/app owner or representative but were unable to find a means to contact them.

### The infringement is a:

- [X] website
- [ ] app or other

### Please drag and drop or attach the screenshot showing the map without proper attribution here.

![Screenshot 2024-03-09 at 15 08 04](https://github.com/openstreetmap/tile-attribution/assets/162812488/0fd97862-f172-48e5-9b0b-b45768532222)

### Where it is happening?

https://mon-cadastre.fr/

**Comment by matkoniecz** [comment]
@peclet1248 Are you planning to send message based on https://osmfoundation.org/wiki/Licence/Attribution_Reminder_Templates ?

**Comment by peclet1248** [comment]
@matkoniecz yes I will send the message today based on your template and I will updated the post after

**Comment by matkoniecz** [comment]
Please also post comment - I will update label then

**Comment by matkoniecz** [comment]
Oh, I see it was edited already. Lets wait then.

Feel free to ping here in on 17 March or later if I will not remember to check it myself.

**Comment by 2803media** [comment]
@matkoniecz, do this repo also flag "commercial" websites which using tiles from osm server (with or without attribution) or only those without a proper attribution (I may have another in my bag ;) )

**Comment by matkoniecz** [comment]
@2803media Neitehr readme nor https://operations.osmfoundation.org/policies/tiles/ has restriction on commercial use.

So only those without a proper attribution, but also non-commercial ones.

**Comment by peclet1248** [comment]
hi @matkoniecz i just check just now and still no response from my email and still no attribution on the website

**Comment by matkoniecz** [comment]
checked, with cache refresh, the still have "© Mon cadastre 2023" with no mention about actual source of map they use in background

![screen](https://github.com/openstreetmap/tile-attribution/assets/899988/73c4c19b-e9f2-4463-9a3a-1e13052a5051)

see https://github.com/openstreetmap/tile-attribution/commit/240a387f0ae41fe7cbec036f8c57bb88a0d8ca60

note that for technical reason block takes some time to be applied - and not apply in some exotic browser configurations

--------------------

thanks for reporting it!

**Comment by Firefishy** [comment]
Block once accepted may take up to 24 hours to apply.

**Comment by peclet1248** [comment]
We've got a clever one ;) He switches tiles from osm to https://tiles.urbanease.io (still OSM but I guess self hosted) and still no attribution of course. For information mon-cadastre and urbanease.io are managed by the same firm.

![Screenshot 2024-03-19 at 11 09 34](https://github.com/openstreetmap/tile-attribution/assets/162812488/9b4b75f9-a481-4aae-af39-84134ac3e197)

**Comment by matkoniecz** [comment]
Yes, in such cases further action can be taken.

But obviously we cannot access that server and easily terminate their tile access.

**Comment by peclet1248** [comment]
> further action can be taken

What kind of action do you think about?

**Comment by Jean-BaptisteC** [comment]
I add a link to issue on OSM france attributions repository https://github.com/osm-fr/attributions/issues/338

**Comment by NathanBnm** [comment]
This has been resolved today. Could you please unblock them? Apparently they weren't informed they were blocked.

**Comment by matkoniecz** [comment]
> Where did you send it?

> By email at [support@urbanease.io](mailto:support@urbanease.io) because the contact form on the website is not working

**Comment by Starmania** [comment]
They finally added the attribution note !

<img width="1853" height="1017" alt="Image" src="https://github.com/user-attachments/assets/ee591115-9133-442a-804b-cc02605696ac" />
